charts/topolvm/values.yaml

```diff
@@ -735,3 +735,11 @@ priorityClass:
 snapshot:
   # snapshot.enabled -- Turn on the snapshot feature.
   enabled: true
+
+crds:
+  # This option makes it so that the "helm.sh/resource-policy": keep
```

Comment:
This comment should be prefixed with `crds.keep -- ` so that it also appears as the comment in the generated `charts/topolvm/README.md`.